app/src/main/java/com/example/bigowlapp/activity/SignUpPageActivity.java

```diff
@@ -62,7 +62,9 @@ protected void initialize() {
                 this.userPhone.setError("please enter a phone number");
                 this.userPhone.requestFocus();
             } else {
-                signUpViewModel.createUser(email, pass, userPhone, firstName, lastName)
+                //String userPhone2 = userPhone.replaceFirst("(\\d{3})(\\d{3})(\\d+)", "($1) $2-$3");  //Format to be used?
+                String userPhone2 = userPhone.replaceAll("[^+0-9]", "");
```

Comment:
rather than `userPhone2`, a more descriptive name of what you did might actually be better to indicate the fact its a number without the dashes.

Also this would be a [Rename Variable](https://refactoring.com/catalog/renameVariable.html) refactor, so please indicate that in the commit comment.